packages/react-components/react-swatch-picker-preview/src/contexts/swatchPicker.ts

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import {
+  SwatchPickerProps,
+  SwatchPickerState,
+  SwatchPickerOnSelectionChangeData,
+  SwatchPickerOnSelectionChangeEvent,
+} from '../components/SwatchPicker/SwatchPicker.types';
+
+/**
+ * The context through which individual color controls communicate with the picker.
+ */
+export type SwatchPickerContextValue = Pick<
```

Comment:
I put context types to this file but maybe all types should be in Component.types.ts file.